### DISCUSSION TITLE
Plot Minirocket kernels

### DISCUSSION THREAD

**Discussion by PaulCharp** [alan-turing-institute/sktime#4659] [Q&A]
Hello Everyone ! 

My first step is to extract features using MinirocketMultivariate. Then, I do classification with a Linear Discreminant Analysis (LDA). From my LDA I get the list of the most predictive features. I was wondering whether it is possible to plot the Minirocket's kernels associated to those  features ? 

Thank you !

**Comment by fkiraly** [comment]
The "default" answer to this would be, use `get_fitted_params` to inspect the fitted model, but in this case I don't think that will give you what you want (because the kernels are not registered as fitted features).

What you could check if `self.parameters` has anything to reconstruct this, on a superficial glance it is not too clear to me whether it is encoded, and if yes how (as it is not documented).

Would be great if you could confirm if that's useful, @PaulCharp, because then it would be a quick fix!

FYI @angus924 - as a developer note, if you want to enable users to see fitted parameters, you need to write them to attributes ending in an underscore per `scikit-learn` convention.

**Reply by PaulCharp** [reply]
Hi @fkiraly, I haven't found anything in self.parameters that could be use to reconstruct the kernels.
Thanks for your fast answer.
